Makefile

```diff
@@ -331,6 +339,11 @@ else ifeq ($(JULIA_BUILD_MODE),debug)
 	$(INSTALL_M) $(build_private_libdir)/sys-debug.$(SHLIB_EXT) $(DESTDIR)$(private_libdir)
 endif
 
+	# Cache stdlibs
+	@$(call PRINT_JULIA, $(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no $(JULIAHOME)/contrib/cache_stdlibs.jl)
+	# CI uses `--check-bounds=yes` which impacts the cache flags
+	@$(call PRINT_JULIA, $(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes $(JULIAHOME)/contrib/cache_stdlibs.jl)
+
```

Comment:
Another reason to eagerly precompile is to avoid having testing processes suddenly try to precompile these files multiple times.  However, we will still run into multiple processes trying to smash these out if someone does a `Base.runtests()` from within the development directory (before running `make install`), right?  Perhaps it's better to add these into a function that gets invoked from within Julia itself upon the running of `Base.runtests()`?  `Pkg.test()` will already precompile everything it needs to, so I don't think we need to worry about user package tests running into this problem.